commands/permissions.go

```diff
@@ -4,35 +4,44 @@
 package commands
 
 import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+	"text/tabwriter"
+
 	"github.com/mattermost/mattermost-server/v5/model"
 
 	"github.com/mattermost/mmctl/client"
-	"github.com/mattermost/mmctl/printer"
 
 	"github.com/spf13/cobra"
 )
 
+const flagAncillaryPermissions = "ancillary"
```

Comment:
Is there a reason to have this flag name in a `const`? Flag names should not change to maintain the CLI's API, so it doesn't look like something that will change frequently

Reply:
No, it wasn't about it changing, it was just that I had to repeat the string 6 times so far and didn't want to make a typo.

Reply:
Actually, after some thought I've decided to remove the ancillary flag entirely in favour of always automatically adding and removing ancillary permission to and from `sysconsole_*` permissions.